lib/DiskSpdParser.pm

```diff
@@ -71,53 +201,32 @@ sub parse_cmd_line($$)
     $cmd_line =~ /-o(\d+)/;
     my $ios_per_thread = $1;
     
-    $stats_ref->{'QD'} = $ios_per_thread * $num_threads;
+    $target_stats{'Total'}{'QD'} = $ios_per_thread * $num_threads;
        
-    if( $cmd_line =~ /-Z.*?([^\\]*)\.bin/ )
+    #search for the format of Storscore's pre-generated entropy files
+    if( $cmd_line =~ /-Z.*?([^\\]*)_pct_comp\.bin/ )
```

Comment:
My intent was to preserve backwards compatibility.  I'm not sure why I made that change -- I reverted it and it works just fine.